sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/Utility.java

```diff
@@ -191,4 +192,17 @@ public static String getPageETag(List<MatchConditions> matchConditionsList, Atom
 
         return nextPageETag;
     }
+
+    // Convert the Map<String, String> to a filter string
+    public static List<String> getTagsFilterInString(Map<String, String> tagsFilter) {
+        List<String> tagsFilters;
+
+        if (tagsFilter != null) {
+            tagsFilters = new ArrayList<>();
+            tagsFilter.forEach((key, value) -> tagsFilters.add(key + "=" + value));
```

Comment:
Should a filter be created if the key or value is `null`?

Reply:
Good catch! A filter should not be created when kv is null